spec/unit/interfaces_spec.rb

```diff
@@ -41,6 +41,16 @@
     fact = Facter.fact("macaddress_net0".intern)
     fact.value.should eq("00:12:34:56:78:90")
   end
+
+  it "should support ipv6 address starting by [180ef]" do
+    Facter.fact(:kernel).stubs(:value).returns("Linux")
+    Facter::Util::IP.stubs(:exec_ifconfig)
+      .returns(my_fixture_read('ifconfig_net_tools_1.60_v6.txt'))
```

Comment:
Apparently ruby 1.8.7 needs a line continuation character (`\`) at the end of line 47 (or lines 47 and 48 to be collapsed to one line).